modules/@apostrophecms/asset/lib/webpack/postcss-replace-viewport-units-plugin.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @type {import('postcss').PluginCreator}
+ */
+module.exports = (opts = {}) => {
+  return {
+    postcssPlugin: 'postcss-replace-viewport-units-plugin',
+    Once (root, postcss) {
+      root.walkRules(rule => {
+        const declsWithContainerQueryRelativeUnits = [];
+
+        rule.walkDecls(decl => {
+          if (!decl.value.includes('vh') && !decl.value.includes('vw')) {
+            return;
+          }
+
+          const declWithContainerQueryRelativeUnits = decl.clone({
```

Comment:
can we change this name to `containerQueryDecl` instead? I had to work hard to spot the difference between `declWithContainerQueryRelativeUnits` and `declsWithContainerQueryRelativeUnits`

Reply:
used `declsToCopy` and `clonedDeclWithContainerQueryUnits`